web/ui/react-app/src/DataTable.tsx

```diff
@@ -59,45 +64,69 @@ class DataTable extends PureComponent<QueryResult> {
 
     let rows: ReactNode[] = [];
     let limited = false;
-    switch(data.resultType) {
+    switch (data.resultType) {
       case 'vector':
-        rows = (this.limitSeries(data.result) as InstantSample[])
-          .map((s: InstantSample, index: number): ReactNode => {
-            return <tr key={index}><td><SeriesName labels={s.metric} format={false}/></td><td>{s.value[1]}</td></tr>;
-          });
+        rows = (this.limitSeries(data.result) as InstantSample[]).map(
+          (s: InstantSample, index: number): ReactNode => {
+            return (
+              <tr key={index}>
+                <td>
+                  <SeriesName labels={s.metric} format={false} />
+                </td>
+                <td>{s.value[1]}</td>
+              </tr>
+            );
+          }
+        );
         limited = rows.length !== data.result.length;
         break;
       case 'matrix':
-        rows = (this.limitSeries(data.result) as RangeSamples[])
-          .map((s, index) => {
-            const valueText = s.values.map((v) => {
+        rows = (this.limitSeries(data.result) as RangeSamples[]).map((s, index) => {
+          const valueText = s.values
+            .map(v => {
```

Comment:
Ah interesting, here it did the opposite and put `.map()` at the beginning of the line where it was previously at the end...